src/scripts/playlists.js

```diff
@@ -1,195 +1,202 @@
-                        reloadItems();
-                    });
-                }
+            if (btnNextPage) {
+                btnNextPage.addEventListener('click', function () {
+                    if (userSettings.libraryPageSize() > 0) {
+                        query.StartIndex += query.Limit;
+                    }
+                    reloadItems();
+                });
+            }
 
-                var btnChangeLayout = view.querySelector('.btnChangeLayout');
+            const btnPreviousPage = view.querySelector('.btnPreviousPage');
 
-                if (btnChangeLayout) {
-                    btnChangeLayout.addEventListener('layoutchange', function (e) {
-                        var layout = e.detail.viewStyle;
-                        getPageData(view).view = layout;
-                        libraryBrowser.saveViewSetting(getSavedQueryKey(view), layout);
-                        onViewStyleChange();
-                        reloadItems();
-                    });
-                }
+            if (btnPreviousPage) {
+                btnPreviousPage.addEventListener('click', function () {
+                    if (userSettings.libraryPageSize() > 0) {
+                        query.StartIndex = Math.max(0, query.StartIndex - query.Limit);
+                    }
+                    reloadItems();
+                });
+            }
 
-                libraryBrowser.saveQueryValues(getSavedQueryKey(view), query);
-                hideLoadingMessage();
-            });
-        }
+            const btnChangeLayout = view.querySelector('.btnChangeLayout');
 
-        var data = {};
-        view.addEventListener('viewbeforeshow', function () {
-            reloadItems();
-        });
-        view.querySelector('.btnNewPlaylist').addEventListener('click', function () {
-            require(['playlistEditor'], function (playlistEditor) {
-                var serverId = ApiClient.serverInfo().Id;
-                new playlistEditor().show({
-                    items: [],
-                    serverId: serverId
+            if (btnChangeLayout) {
+                btnChangeLayout.addEventListener('layoutchange', function (e) {
+                    const layout = e.detail.viewStyle;
+                    getPageData(view).view = layout;
+                    libraryBrowser.saveViewSetting(getSavedQueryKey(view), layout);
+                    onViewStyleChange();
+                    reloadItems();
                 });
+            }
+
+            libraryBrowser.saveQueryValues(getSavedQueryKey(view), query);
+            hideLoadingMessage();
+        });
+    }
+
+    const data = {};
+    view.addEventListener('viewbeforeshow', function () {
+        reloadItems();
+    });
+    view.querySelector('.btnNewPlaylist').addEventListener('click', function () {
+        import('playlistEditor').then(({default: playlistEditor}) => {
+            const serverId = ApiClient.serverInfo().Id;
+            new playlistEditor.showEditor({
```

Comment:
`showEditor` is `default`
```suggestion
            new playlistEditor({
```
But I cannot test this - it seems that this file isn't used.